mysql-test/suite/rpl/t/percona_bug_ps3885.test

```diff
@@ -0,0 +1,26 @@
+#
```

Comment:
Just in case, start this MTR test case with `--repeat=2` and check if the second run passes.

Also, to make sure that there is no nondeterminism in slaves catching up masters, I usually run MTR under big stress:
```
./mtr --debug-server --parallel=24 --repeat=16 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885 percona_bug_ps3885
```